website/docs/cdktf/release/index.mdx

```diff
@@ -17,8 +17,25 @@ CDKTF includes upgrade guides for releases that have breaking changes.
 - [Upgrading to v0.7](/cdktf/release/upgrade-guide-v0-7)
 - [Upgrading to v0.6](/cdktf/release/upgrade-guide-v0-6)
 
-## Enable Features
+## Features Flags for 0.11
 
-CDKTF uses feature flags to enable potentially breaking behaviors in a release. Feature flags are disabled by default, so existing projects that do not specify the flag will continue to work as expected with later CDKTF releases. New projects created using `cdktf init` include flags enabling all features available in the release that created the project.
+CDKTF uses feature flags to enable potentially breaking behaviors in a release. You can use them to change the logic when synthesizing code or CLI features, but they cannot change the logic of the provider generation.
```

Comment:
Moving this up, as I think it can be combined with the introduction. But question @DanielMSchmidt - the sentence about how the default is to be set to `true` feels like it directly contradicts the paragraph below this about how how they're disabled by default... help :) 

```suggestion
CDKTF uses feature flags to enable potentially breaking behaviors in a release. You can use them to change the logic when synthesizing code or CLI features, but they cannot change the logic of the provider generation. A feature flag will be introduced in a release, adding it with a default `true` value to the templates used in `cdktf init`. We will print a warning if it's false or not set. Feature flags will be removed in the next minor release, and the standard behavior is the feature flag set to `true`. 
```